### ISSUE TITLE
timeout*1000 is harmful!

### ISSUE THREAD

**Issue by Demetrio92** [r-lib/pingr#7]
I don't know whether this also affects windows users, but the sys call to `ping` on linux is executed with `timeout * 1000` passed to `-W` argument.  My `man ping` (ubuntu 16.04) says:
```
       -W timeout
              Time to wait for a response, in **seconds**. The option affects only timeout in 
              absence of  any responses, otherwise ping waits for two RTTs.
```

Therefore calling `ping('lalala.com')` dead-locks the R session (ok, actually, only affects Rstudio, and not the terminal, but nonetheless). 
The documentation does not say anything about the measurement units, and the default for `timeout` argument is set to `1`, 

IMO, two things should be changed: 
1. Documentation should be clear about the units
2. The default should be max 10 seconds

**Comment by gaborcsardi** [comment]
The dead lock will be solved by #8. Sadly, it seems that different ping programs take different units, so I would need to rewrite the ICMP ping in C: #11.

Btw. there is a new `is_up()` function, and I guess people would need that more often than `ping()`.

**Comment by gaborcsardi** [comment]
Nevertheless if looks like Linux has the timeout in seconds, so I can do that change already now.
